odoo90/scripts/build-image.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# With a little help from my friends
+. /usr/share/vx-docker-internal/ubuntu-base/library.sh
+. /usr/share/vx-docker-internal/odoo80/library.sh
+
+# Let's set some defaults here
+ARCH="$( dpkg --print-architecture )"
+NODE_UPSTREAM_REPO="deb http://deb.nodesource.com/node_5.x trusty main"
+NODE_UPSTREAM_KEY="https://deb.nodesource.com/gpgkey/nodesource.gpg.key"
+WKHTMLTOX_URL="http://download.gna.org/wkhtmltopdf/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-${ARCH}.tar.xz"
+ODOO_DEPENDENCIES="git+https://github.com/vauxoo/odoo@8.0 \
+                   git+https://github.com/vauxoo/server-tools@8.0 \
+                   git+https://github.com/vauxoo/addons-vauxoo@8.0 \
+                   git+https://github.com/vauxoo/odoo-venezuela@8.0 \
+                   git+https://github.com/vauxoo/pylint-odoo@master"
+                   # git+https://github.com/vauxoo/odoo-mexico-v2@8.0 \
```

Comment:
Yes, as far as i know, the methodology to follow for the 9.0 image is to temporarily copy the 8.0 branch. This will certainly fail, but will be fixed in a future PR, correct me if i'm wrong, @nhomar.